### DISCUSSION TITLE
Превентивная очистка браузерной строки ввода

### DISCUSSION THREAD

**Discussion by alex93skr** [k-ivan/visual-bookmarks-chrome#142] [Ideas]
Можно ли сделать фичу, что бы вместо 

![image](https://github.com/k-ivan/visual-bookmarks-chrome/assets/49789700/148cd80d-983f-4509-9f55-60886a589693)

строка изначально очищалась и отображалась пустой

![image](https://github.com/k-ivan/visual-bookmarks-chrome/assets/49789700/6c95c514-c94d-4baa-813e-6eaf6af855dd)

Нижний поиск у меня настроен на яндекс, а браузеная строка на гугл - и когда в нем нужно что то найти, оно не всегда самоочищается при клике по нему. И получается поисковой запрос типа: 
"chrome-extension://jdbgjlehkajddoapd**как помыть кошку**gpdjmlpdalfnenf/newtab.html"

**Comment by k-ivan** [comment]
Чисто теоретически в chrome/edge такое можно провернуть, но для других хромиум подобных браузеров могут быть свои префиксы в в начале пути (chrome://newtab или edge://newtab), все это надо отслеживать(у каждого браузера свой путь), а если сделать только для официально поддерживаемых, то сломается в остальных(где расширение работало). И так же не уверен, работает ли этот путь в фаерфоксе, потому что хочу портировать расширение в фаерфокс. Поэтому не уверен, что это стоит того, учитывая, что можно упереться в то, что сломается поведение в других браузерах. А костылить (в одном так, в другом не так) не хочется.
